tron/core/job_scheduler.py

```diff
@@ -85,6 +85,26 @@ def schedule(self):
             return
         self.create_and_schedule_runs()
 
+    def update_from_job_scheduler(self, job_scheduler):
+        """ Update a job scheduler by copying another. """
+        old_job = self.get_job()
+        new_job = job_scheduler.get_job()
+
+        old_job.update_from_job(new_job)
+
+        # Since job upadting only copies equality attributes (defined in the Job
+        # class), we need to now enable or disable the job depending on the
+        # new job says so.
+        if old_job.enabled is not new_job.enabled and \
+           old_job.config_enabled is not new_job.config_enabled:
+            if new_job.enabled:
```

Comment:
something is off with indentation here or in previous line

Reply:
You're right. I'll update it to follow the Python style guide.